src/InputSheath/inputSheath.style.js

```diff
@@ -2,10 +2,10 @@ import styled from "styled-components";
 import { error, errorBackground } from "../Colors";
 
 const InputStyles = styled.div`
+  position: relative;
```

Comment:
@lanwen the previous `position: relative;` is obsolete because of the existence of this one.